saylua/models/user.py

```diff
@@ -95,38 +90,6 @@ def hash_password(cls, password, salt=None):
     def check_password(cls, user, password):
         return cls.hash_password(password, user.phash) == user.phash
 
-    # @classmethod
-    # @ndb.transactional(xg=True)
-    # def transfer_currency(cls, from_key, to_key, cc=0, ss=0):
```

Comment:
Why remove these? They will need to be re-implemented in the future as far as I understand it.